File: ACTIVIDADESDELADSO2928797/Trabajo_en_clase/dart/24-05-2024/EJERCICIO.dart
import 'dart:io';

void main(){
  print ('Ingrese su nombre, por favor:');
  String? n = stdin.readLineSync();

  print ('Ingrese su apellido, por favor:');
  String? ap = stdin.readLineSync();

  print ('Ingrese el primer valor, por favor:');
  int n1 = int.parse(stdin.readLineSync()!);
  
  print ('Ingrese el segundo valor, por favor:');
  int n2 = int.parse(stdin.readLineSync()!);

  print ('Ingrese el tercer valor, por favor:');
  int n3 = int.parse(stdin.readLineSync()!);

  int sum = n1 + n2 + n3;
  var prom = sum / 3;

  print( "Señor(a) $n $ap");
  print("Su promedio es igual a $prom"); 

}

File: ACTIVIDADESDELADSO2928797/Trabajo_en_clase/Ruby/24-05-2024/ejercicio.rb
puts "Ingrese su nombre, por favor: ";
n = gets.chomp;

puts "Ingrese su apellido, por favor: ";
ap = gets.chomp;

puts "Ingrese el primer valor, por favor:";
n1 = gets.chomp.to_i;

puts "Ingrese el segundo valor, por favor: ";
n2 = gets.chomp.to_i;

puts "Ingrese el tercer valor, por favor";
n3 = gets.chomp.to_i;

sum = n1 + n2 + n3;
prom = sum/3;

print "Señor(a) #{n} #{ap}, ";
print "su promedio es igual a: #{prom}"; 
